pkg/artifactory/provider.go

```diff
@@ -126,8 +126,8 @@ func Provider() *schema.Provider {
 				Type:        schema.TypeString,
 				Optional:    true,
 				Sensitive:   true,
-				DefaultFunc: schema.MultiEnvDefaultFunc([]string{"ARTIFACTORY_ACCESS_TOKEN", "JFROG_ACCESS_TOKEN"}, ""),
-				Description: "This is a access token that can be given to you by your admin under `Identity and Access`",
+				DefaultFunc: schema.MultiEnvDefaultFunc([]string{"ARTIFACTORY_ACCESS_TOKEN", "JFROG_ACCESS_TOKEN"}, nil),
+				Description: "(Optional) This is a access token that can be given to you by your admin under `Identity and Access`. Default to 'nil'.",
```

Comment:
`nil` means nothing to the end user. You have to explain the behavior they can expect. `If not set, the API_KEY will be used` For example.